tink-docker/main.go

```diff
@@ -79,11 +79,28 @@ func parseCmdLine(cmdLines []string) (cfg tinkConfig) {
 // downloadFile will download a url to a local file. It's efficient because it will
 // write as it downloads and not load the whole file into memory.
 func downloadFile(filepath string, url string) error {
+	// As all functions in the LinuxKit services run in parallel, ensure that we can fail
+	// successfully until we accept that networking is actually broken
+
+	var maxRetryCount int
+	var timeOut time.Duration
+	maxRetryCount = 10
+	timeOut = time.Millisecond * 500 // 0.5 seconds
+	var resp *http.Response
+	var err error
+
+	// Retry this task
+	for retries := 0; retries < maxRetryCount; retries++ {
 	// Get the data
-	resp, err := http.Get(url)
-	if err != nil {
+	resp, err = http.Get(url)
```

Comment:
Could you try running `go fmt` on this file? It looks like the formatting had a bad time.

Reply:
Done.

Reply:
Thank you :)